pkg/manager/member/tikv_member_manager.go

```diff
@@ -511,6 +511,11 @@ func getNewTiKVSetForTidbCluster(tc *v1alpha1.TidbCluster, cm *corev1.ConfigMap)
 				ContainerPort: int32(20160),
 				Protocol:      corev1.ProtocolTCP,
 			},
+			{
+				Name:          "status",
```

Comment:
We have to add a parameter, e.g. `enableNamedStatusPort`, in the `spec.tikv` of TidbCluster CR and default to `false`, otherwise, this change will trigger TiKV cluster rolling update.